packages/web-react/src/components/Button/demo/ButtonDemoFactory.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import DocsSection from '../../../../docs/DocsSections';
-import { ActionButtonColors, EmotionColors, Sizes } from '../../../constants';
+import { ComponentButtonColors, EmotionColors, Sizes } from '../../../constants';
 import { ButtonBaseProps } from '../../../types';
 import { Icon } from '../../Icon';
 import Button from '../Button';
 
 const ButtonDemoFactory = ({ ...props }: ButtonBaseProps) => {
   const sizes = Object.values(Sizes);
-  const actionButtonColors = Object.values(ActionButtonColors);
+  const actionButtonColors = Object.values(ComponentButtonColors);
```

Comment:
rename this const please

Reply:
I mean `actionButtonColors`, i think `buttonColors` is enough or `componentButtonColors`